### DISCUSSION TITLE
Purpose of "objects.githubusercontent.com"

### DISCUSSION THREAD

**Discussion by EzraAriwomoi** [community/community#137510] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey everyone,

I'm trying to find more information regarding the purpose objects.githubusercontent.com. Does anyone else have more info on its purpose? I understand that raw.githubusercontent.com is used to deliver unprocessed versions of files hosted in a repo.

If anyone has further information, I would deeply appreciated!

**Comment by Marcelo09102000-curry** [comment]
![AddText_08-01-02 10 27](https://github.com/user-attachments/assets/b32ff054-a49e-490d-a189-27127d99d3cb)
m1 Channel not subscribe

![AddText_08-01-03 03 45 no signal](https://github.com/user-attachments/assets/3d53bf31-8d0e-41a7-9e01-c2fea8a3d4d2)
m2  No signal level

![AddText_08-01-03 06 47](https://github.com/user-attachments/assets/5d2ae2e3-006f-449f-b412-affa3a16467e)
m3 corrupt majority

![AddText_08-01-03 14 18](https://github.com/user-attachments/assets/b744e9ee-a265-484d-90a3-21ccacb755a8)
m4 Format unsupported

![AddText_08-01-03 10 28](https://github.com/user-attachments/assets/ef63f584-60fa-45d6-a5c0-3a48ba1f0c0b)
m5 not file found

![AddText_08-01-05 09 04](https://github.com/user-attachments/assets/911b160f-2d01-46c2-bd7b-ad3faf5d7b69)
m6 Program unavailable

![AddText_08-01-05 21 49](https://github.com/user-attachments/assets/c2d786dd-8870-41a0-98bf-acbb9a9508e9)
m7 Program crushed

**Comment by Dev-axay18** [comment]
`objects.githubusercontent.com` is used by **GitHub to store and deliver large files** that are managed through **Git LFS (Large File Storage)**. It’s different from `raw.githubusercontent.com`, which is used to show raw text or code files. So if a repo uses Git LFS for big files like videos or datasets, those files are downloaded from `objects.githubusercontent.com`.
